incydr/cli/__init__.py

```diff
@@ -1,21 +1,71 @@
+import os
+
 import click
 from rich.console import Console
 
-from incydr import Client
+from incydr.cli.utils import get_user_project_path
+
+ERROR_LOG_FILE_NAME = "incydr_cli.log"
 
 console = Console()
 
+
+def log_level_callback(ctx, param, value):
+    """callback to update log_level environment variable"""
+    if not value:
+        return
+    os.environ.update({"INCYDR_LOG_LEVEL": value})
+    return value
+
+
+def log_file_callback(ctx, param, value):
+    """callback to update log_file and log_stderr environment variable"""
+    if not value:
+        return
+    os.environ.update({"INCYDR_LOG_FILE": value})
+    return value
+
+
+def log_stderr_callback(ctx, param, value):
+    """callback to update log_stderr environment variable"""
+    if not value:
+        os.environ.update({"INCYDR_LOG_STDERR": "FALSE"})
+        return
+    os.environ.update({"INCYDR_LOG_STDERR": "TRUE"})
+    return value
+
+
+# TODO: do we want these options to be hidden from --help (add hidden=True to each def)
```

Comment:
^

Reply:
I'm ok with them being exposed on every command, they should be always at the bottom of the options list by --help though, so we should make sure they're the first option decorator applied to each command function.

Reply:
Do you mean the last decorator?

Reply:
Ohhh, yea I think it is in that position on all the commands.